### PR TITLE
LPS-193163 - Automation | Forms|  Master

### PR DESCRIPTION
**Pull request by thaiscabrals**
**Bug:** [LPS-176235](https://liferay.atlassian.net/browse/LPS-176235)
**Automation:** [LPS-193163](https://liferay.atlassian.net/browse/LPS-193163)
**Test:** UserGuestCanVerifyEntriesPrivate

### CODE REVIEW

**Comment by thaiscabrals** [comment]
ci:test:sf

**Comment by thaiscabrals** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsUploadField.testcase [hunk 1]
```diff
@@ -984,4 +984,88 @@ definition {
...
+			PagesAdmin.configurePage(
+				columnName = "1 Column",
+				pageName = "Form Page",
+				sitePageType = "Widget Page");
+
+			Navigator.gotoPage(pageName = "Form Page");
+
+			Portlet.addPG(
+				pageName = "Form Page",
+				portletName = "Form");
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsUploadField.testcase:1024]: @thaiscabrals can you do this via JSON so our tests run faster?

**Review by patriciajeanperez — CHANGES_REQUESTED** [review]
@thaiscabrals Test looks good, but can we use JSON macros for steps such as adding a page and a widget to page? That will make our tests faster. Thanks!

**Comment by thaiscabrals** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**